src/iceberg/json_internal.cc

```diff
@@ -173,12 +171,17 @@ void SetOptionalField(nlohmann::json& json, std::string_view key,
   }
 }
 
+std::string DumpJsonNoExcept(const nlohmann::json& json) {
+  return json.dump(-1, ' ', false, nlohmann::detail::error_handler_t::ignore);
```

Comment:
```suggestion
  return json.dump(/*indent=*/-1, /*indent_char=*/' ', /*ensure_ascii=*/false, nlohmann::detail::error_handler_t::ignore);
```

For better readability

Reply:
What about renaming it to `SafeDumpJson`?

Reply:
Fixed, thanks.